geomstats/geometry/grassmannian.py

```diff
@@ -117,7 +118,7 @@ def _squared_dist(point_a, point_b, metric):
     _ : array-like, shape=[...,]
         Geodesic distance between point_a and point_b.
     """
-    return metric.private_squared_dist(point_a, point_b)
+    return metric._squared_dist(point_a, point_b)
```

Comment:
Ah nice! How did you solve this? The introduction of the very ugly "private_squared_dist" was to get the automatic differentiation with custom gradient work with all backends.

Maybe it was not needed anymore since we dropped TF?

Reply:
here is really just naming. I'm representing private by `_`. But nothing changes from a logical perspective.